scoring/templates/scoring/council.html

```diff
@@ -94,11 +93,10 @@ <h3 class="mb-3">About the scoring</h3>
             <div class="mb-5">
                 <div class="table-header mb-4">
                     <h3>
-                        Section
+                        Scorecard section performance
                     </h3>
                     <!-- TODO: New text -->
```

Comment:
Don’t think we need this `<!-- TODO` any more. Should do a sweep over all the files this branch has touched, to check whether there are any other leftover `TODO`s now the text they were referring to has been updated.